src/raftstore/store/store.rs

```diff
@@ -2389,13 +2423,19 @@ impl<T: Transport, C: PdClient> mio::Handler for Store<T, C> {
                 info!("{} receive quit message", self.tag);
                 event_loop.shutdown();
             }
+            // TODO: Unified split region message.
             Msg::SplitCheckResult {
                 region_id,
                 epoch,
                 split_key,
             } => {
                 info!("[region {}] split check complete.", region_id);
-                self.on_split_check_result(region_id, epoch, split_key);
+                self.on_prepare_split_region(
+                    region_id,
+                    epoch,
+                    keys::origin_key(split_key.as_slice()).to_vec(),
```

Comment:
pd AskSplit use different split_key type now?

Reply:
Actully the `split_key` here is a data key, AskSplit requires encoded key, so convert it by `keys::origin_key`.